### PR TITLE
Add smart contract submission functionality for Purchase Order

### PR DESCRIPTION
**Pull request by agunde406**
Updates AppAuthHandler and docker-compose files to build/submit the Grid Purchase Order smart contract

### CODE REVIEW

File: daemon/src/splinter/app_auth_handler/sabre.rs [hunk 1]
```diff
@@ -69,6 +72,10 @@ const SCHEMA_CONTRACT_NAME: &str = "grid-schema";
 #[cfg(feature = "location")]
 const LOCATION_CONTRACT_NAME: &str = "grid-location";
 
+// Purchase Order constants
+#[cfg(feature = "location")]
```
Comment (vaporos) [inline daemon/src/splinter/app_auth_handler/sabre.rs:76]: This should probably be purchase-order

Comment (agunde406) [reply daemon/src/splinter/app_auth_handler/sabre.rs:76]: whoops yep

Comment (agunde406) [reply daemon/src/splinter/app_auth_handler/sabre.rs]: fixed

File: contracts/purchase_order/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Cargill Incorporated
```
Comment (shannynalayna) [inline contracts/purchase_order/Dockerfile:1]: s/2020/2021

Comment (agunde406) [reply contracts/purchase_order/Dockerfile]: fixed

**Review by shannynalayna — APPROVED** [review]

**Pull request merged**